bolt-sidecar/src/telemetry/metrics.rs

```diff
@@ -64,7 +64,6 @@ impl ApiMetrics {
     }
 
     /// Counters ----------------------------------------------------------------
-
```

Comment:
this line was supposed be a sort of divider instead of a doc comment.
Could you try re-adding the removed line and changing the comment to `//` instead of `///`?

If it still auto-formats to collapse it like this it's also fine, it's just a tedious nit :D

Reply:
haha, I'll take the feedback